src/main/java/coms/w4156/moviewishlist/utils/StreamingConstants.java

```diff
@@ -0,0 +1,70 @@
+package coms.w4156.moviewishlist.utils;
```

Comment:
I created this streaming constants file because I didn't like all the hardcoded strings I was using

Reply:
A lot of the diffs show me converting from hard-coded to using the constant instead